builtin/providers/aws/resource_aws_ses_receipt_rule.go

```diff
@@ -0,0 +1,730 @@
+							Type:     schema.TypeString,
+							Required: true,
+						},
+
+						"position": &schema.Schema{
+							Type:     schema.TypeInt,
+							Required: true,
+						},
+					},
+				},
+				Set: func(v interface{}) int {
+					var buf bytes.Buffer
+					m := v.(map[string]interface{})
+					buf.WriteString(fmt.Sprintf("%s-", m["topic_arn"].(string)))
+					buf.WriteString(fmt.Sprintf("%d-", m["position"].(int)))
+
+					return hashcode.String(buf.String())
+				},
+			},
+
+			"stop_action": &schema.Schema{
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"scope": &schema.Schema{
+							Type:     schema.TypeString,
+							Required: true,
+						},
+
+						"topic_arn": &schema.Schema{
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+
+						"position": &schema.Schema{
+							Type:     schema.TypeInt,
+							Required: true,
+						},
+					},
+				},
+				Set: func(v interface{}) int {
+					var buf bytes.Buffer
+					m := v.(map[string]interface{})
+					buf.WriteString(fmt.Sprintf("%s-", m["scope"].(string)))
+					buf.WriteString(fmt.Sprintf("%s-", m["topic_arn"].(string)))
+					buf.WriteString(fmt.Sprintf("%d-", m["position"].(int)))
+
+					return hashcode.String(buf.String())
+				},
+			},
+
+			"workmail_action": &schema.Schema{
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"organization_arn": &schema.Schema{
+							Type:     schema.TypeString,
+							Required: true,
+						},
+
+						"topic_arn": &schema.Schema{
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+
+						"position": &schema.Schema{
+							Type:     schema.TypeInt,
+							Required: true,
+						},
+					},
+				},
+				Set: func(v interface{}) int {
+					var buf bytes.Buffer
+					m := v.(map[string]interface{})
+					buf.WriteString(fmt.Sprintf("%s-", m["organization_arn"].(string)))
+					buf.WriteString(fmt.Sprintf("%s-", m["topic_arn"].(string)))
+					buf.WriteString(fmt.Sprintf("%d-", m["position"].(int)))
+
+					return hashcode.String(buf.String())
+				},
+			},
+		},
+	}
+}
+
+func resourceAwsSesReceiptRuleCreate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).sesConn
+
+	createOpts := &ses.CreateReceiptRuleInput{
+		Rule:        buildReceiptRule(d, meta),
+		RuleSetName: aws.String(d.Get("rule_set_name").(string)),
+	}
+
+	if v, ok := d.GetOk("after"); ok {
+		createOpts.After = aws.String(v.(string))
+	}
+
+	_, err := conn.CreateReceiptRule(createOpts)
+	if err != nil {
+		return fmt.Errorf("Error creating SES rule: %s", err)
+	}
+
+	d.SetId(d.Get("name").(string))
+
+	return resourceAwsSesReceiptRuleUpdate(d, meta)
+}
+
+func resourceAwsSesReceiptRuleUpdate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).sesConn
+
+	updateOpts := &ses.UpdateReceiptRuleInput{
```

Comment:
Hmm, on second thought, that doesn't actually make any sense, since there could be an `after` change along with some other change, and we do want to trigger an update. I could do a mass change check for every other field, but that seems kind of messy. I'm not sure that it's even a problem to call the update if nothing has changed.